src/main/java/com/griefcraft/listeners/LWCEntityListener.java

```diff
@@ -239,7 +239,7 @@ private void entityCreatedByPlayer(Entity entity, Player player) {
         String autoRegisterType = lwc.resolveProtectionConfiguration(entity.getType(), "autoRegister");
 
         // is it auto protectable?
-        if (!autoRegisterType.equalsIgnoreCase("private") && !autoRegisterType.equalsIgnoreCase("public") && !autoRegisterType.equalsIgnoreCase("donation")) {
+        if (!autoRegisterType.equalsIgnoreCase("private") && !autoRegisterType.equalsIgnoreCase("public") && !autoRegisterType.equalsIgnoreCase("donation") && !autoRegisterType.equalsIgnoreCase("display") && !autoRegisterType.equalsIgnoreCase("supply")) {
```

Comment:
As far as I read the source code, for specific setting, players can protect any entity and only private and password protection have meaning. ([Reference](https://github.com/pop4959/LWCX/blob/master/src/main/java/com/griefcraft/lwc/LWC.java#L755))

Thus, if supply check is not needed, public, donation, display check are also not needed.

For now, I will remove supply check. It may be good to think abount how this check should be after this PR.
In my opinion, these meaningless protection type at this time can work for donkey or something which have inventory if someone add implementation,

Reply:
I just don't care to update the entity protection feature in LWC as it's currently very broken, no one should be using it. So no need to waste time on it 👍